hw1_task2/index.js

```diff
@@ -0,0 +1,21 @@
+function sum(...args) {
+
+    let sum = args.reduce( (sum, e) => sum + e );
```

Comment:
@matveyprupas We run the same reduce function twice. Could we fix that?

Reply:
We can fix it by using a little function for this operation. But it will be the same effectivity.